.github/workflows/cypress.yml

```diff
@@ -0,0 +1,27 @@
+name: E2E test
+
+on:
+  pull_request:
+    branches:
+      - develop
```

Comment:
Shouldn't we test it on push to: trunk and develop?

Reply:
Yes, we should. Good catch. done.
I will raise an issue in wp-cypress-setup lib

Reply:
@cadic do you think should we add this in cypress-wp-setup lib?